desktop/core/src/desktop/js/components/sidebar/HelpDrawerContent.vue

```diff
@@ -22,8 +22,12 @@
       <h2>Help</h2>
     </div>
     <ul>
-      <li v-for="childItem in children" :key="childItem.name">
-        <BaseNavigationItem :item="childItem" @click="hideDrawer">
+      <li v-for="(childItem, index) in children" :key="index">
```

Comment:
Why :key=index? name should be unique...

Reply:
For SidebarSpacerItem there is no name.